### ISSUE TITLE
The loss function is wrong?

### ISSUE THREAD

**Issue by cl5220418** [Epiphqny/PGN-pytorch#1]

**Comment by Epiphqny** [comment]
The loss function does not include the boundary part, if you need that, you
can add it yourself, and uncomment the boundary part in the network file

cl5220418 <notifications@github.com> 于2019年5月10日周五 下午7:23写道：
